packages/synapse-react-client/src/components/SynapseTable/SynapseTable.tsx

```diff
@@ -774,16 +767,25 @@ export class SynapseTable extends React.Component<
                         name: column.name,
                       })}
                     >
-                      <Icon
-                        type={ICON_STATE[columnIndex]}
-                        cssClass={isSelectedIconClass}
-                      ></Icon>
+                      <IconSvg
+                        icon={ICON_STATE[columnIndex]}
+                        wrap={false}
+                        sx={{
+                          color: isSelected
+                            ? 'primary.contrastText'
+                            : 'primary.main',
+                          backgroundColor: isSelected ? 'primary.main' : 'none',
+
+                          '&:hover': {
+                            color: 'primary.contrastText',
+                            backgroundColor: 'primary.600',
+                          },
+                        }}
```

Comment:
MUI's `sx` prop allows us to use certain strings to reference theme variables, like `primary.main` and `primary.contrastText` in this case.